programs/stake/src/stake_state.rs

```diff
@@ -4280,7 +4284,7 @@ mod tests {
         let stake_pubkey = solana_sdk::pubkey::new_rand();
         let rent = Rent::default();
         let rent_exempt_reserve = rent.minimum_balance(std::mem::size_of::<StakeState>());
-        let stake_lamports = rent_exempt_reserve * 3; // Enough to allow half to be split and remain rent-exempt
+        let stake_lamports = rent_exempt_reserve * 3 + MINIMUM_STAKE_DELEGATION * 2; // Enough to allow half to be split and remain rent-exempt
```

Comment:
nit: make this a bit closer to the limits
```suggestion
        let stake_lamports = rent_exempt_reserve * 2 + MINIMUM_STAKE_DELEGATION * 2; // Enough to allow half to be successfully split
```

Reply:
Done!